lib/services/database.go

```diff
@@ -339,6 +460,20 @@ func labelsFromRedshiftCluster(cluster *redshift.Cluster, meta *types.AWS) map[s
 	return labels
 }
 
+// labelsFromElastiCacheCluster creates database labels for the provided
+// ElastiCache cluster.
+func labelsFromElastiCacheCluster(cluster *elasticache.ReplicationGroup, meta *types.AWS, endpointType string, extraLabels map[string]string) map[string]string {
```

Comment:
nit: Seems that cluster var is unused:
```suggestion
func labelsFromElastiCacheCluster(meta *types.AWS, endpointType string, extraLabels map[string]string) map[string]string {
```